ApplicationLibCode/ProjectDataModel/RimColorLegend.h

```diff
@@ -43,6 +43,9 @@ class RimColorLegend : public caf::PdmObject
     RimColorLegend();
     ~RimColorLegend() override;
 
+public:
+    caf::Signal<> legendChanged;
```

Comment:
Is `changed` better than `legendChanged` here?

Reply:
Fixed